go/utils/helper.go

```diff
@@ -0,0 +1,45 @@
+package utils
```

Comment:
Well, utils in go encounters less namespace conflicts than other languages, like we can import like 
```go
import (

utils "github.com/PaddlePaddle/Paddle/go/utils"
masterutils "github.com/PaddlePaddle/Paddle/go/master/utils"

)
```

But indeed the name `utils` is like `data` or `blob` means nothing. Will put to a more meaningful package like in https://github.com/kubernetes/kubernetes/tree/master/pkg/util